tests/test_utils.py

```diff
@@ -48,34 +49,32 @@ def test_cache_folder_default(self):
     def test_get_image_online(self, image_id, tmpdir, mocker):
         """A test for the get_image function"""
 
-        mocker.patch("os.path.exists", return_value=True)
+        mocker.patch("pathlib.Path.exists", return_value=True)
+        fake_img = Image.fromarray(np.zeros((100, 200), dtype=np.uint8))
         mocker.patch(
             "matplotlib.pyplot.imread",
             return_value=np.zeros((100, 200), dtype=np.uint8),
         )
 
```

Comment:
Do we still need the mocking of `matplotlib` ?

Reply:
Nice catch! You're right, we don't need it. Fixed in 6184c22